go/vt/mysqlctl/backup.go

```diff
@@ -104,6 +104,7 @@ func init() {
 
 func registerBackupFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&backupStorageHook, "backup_storage_hook", backupStorageHook, "if set, we send the contents of the backup files through this hook.")
+	_ = fs.MarkDeprecated("backup_storage_hook", "consider using ExternalCompressorCmd/ExternalDecompressorCmd instead.")
```

Comment:
This doesn't seem correct. we should provide the alternate flags (starting with `--`) instead of the internal variable names.
i.e. something like
```
consider using one of the builtin compression algorithms or --external-compressor-cmd and --external-decompressor-cmd instead
```

Reply:
fixed